apps/dash-salesforce-crm/assets/s1.css

```diff
@@ -0,0 +1,335 @@
+body {
```

Comment:
I would give this a quick read [](https://gist.github.com/basham/2175a16ab7c60ce8e001), and just make sure some things are more consistent, such as your units for paddings, margins, font size. I would recommend of letting go of PX in favour of REM in all cases, save for non-scaling properties, such as borders and box shadows.